src/components/PomoDesc/PomoDesc.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useModal } from "../../context/modal-context";
+import "./pomoDesc.css";
+const PomoDesc = () => {
+  const { description } = useModal();
+
+  return (
+    <>
+      <div className="pomodoro">
+        <div className="pomodoro-container">
+          {description.map((pomo) => {
+            return (
```

Comment:
suggestion: you can simply return it without using return statement.

Reply:
yes